cypress/integration/examples/viewport.spec.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+
+context('Viewport', () => {
+  beforeEach(() => {
+    cy.visit('https://example.cypress.io/commands/viewport')
```

Comment:
The URL visited should be the Book Store app URL, not 'https://example.cypress.io/commands/viewport'.